.buildkite/solana-lab.yml

```diff
@@ -0,0 +1,103 @@
+x-default-agents: &default-agents
+  agents:
+    queue: "${AGENT_QUEUE:-default}"
+
+x-test-template: &test-template
+  <<: *default-agents
+  depends_on: ["build"]
+  concurrency: 1
+  concurrency_group: pulumi-testing
+
+  command: |
+    source opsh
+    source .buildkite/git.opsh
+    source .buildkite/lib.opsh
+
+    # setup an ssh agent.  Will need to use '-F $_OPSH_SSH_CONTEXT/config' later for ssh
+    ssh::setup
+
+    # Setup ssh for private repo
+    buildkite-agent secret get GITHUB_TOOLING_KEY | ssh::key::add
+    git::repo-add git@github.com:abklabs/solana-lab.git solana-lab
+    git::repo-add git@github.com:abklabs/tooling.git tooling --ssh-secret GITHUB_TOOLING_KEY
+    git::repos-summary
+    git::repos-clone
+
+    if [[ -n "${SVMKIT_DIR:-}" || -n "${PULUMI_SVMKIT_DIR:-}" ]]; then
+      echo "-- Downloading Pulumi SDK and provider plugin artifacts"
+      buildkite-agent artifact download pulumi-artifacts.tgz .
+      tar zxf pulumi-artifacts.tgz
+      PATH="$PWD/pulumi-svmkit/bin:$PATH"
```

Comment:
This PATH is handled by https://github.com/abklabs/tooling/blob/main/bin/with-local-pulumi-svmkit#L35  .. you shouldn't do this here as well.